R/pmcmc_tools.R

```diff
@@ -0,0 +1,49 @@
+##' Thin results of running [pmcmc()]. This function may be useful
+##' before using [pmcmc_predict()], or before saving pmcmc output to
+##' disk.
+##'
+##' @title Thin a pmcmc chain
+##' @param object Results of running [pmcmc()]
+##'
+##' @param burnin Optional integer number of iterations to discard as
+##'   "burn-in". If given then samples `1:burnin` will be excluded
+##'   from your results. Remember that the first sample represents the
+##'   starting point of the chain. It is an error if this is not a
+##'   positive integer or is greater than the number of samples.
+##'
+##' @param thin Optional integer thinning factor. If given, then every
+##'   `thin`'th sample is retained (e.g., if `thin` is 10 then we keep
```

Comment:
thin'th – love it

Reply:
pity those with a lisp

Reply:
(update here is just that I noticed I'd prematurely started using markdown before we support it)